commands.py

```diff
@@ -0,0 +1,47 @@
+from django.contrib.auth.decorators import permission_required

```

Comment:
I suspect this is leftover from a previous vision of how this would be implemented?